src/vector/rageshakesetup.js

```diff
@@ -44,3 +45,17 @@ function initRageshake() {
 }
 
 initRageshake();
+
+global.mxSendRageshake = function(text, withLogs) {
```

Comment:
Would it be useful to default `withLogs` to `true` if it isn't specified?  There is a bit of a privacy concern that people might be sending logs without realizing it, but it seems to me like the main use case for this function is to send logs.  I'd be fine either way.